projects/storefrontapp-e2e-cypress/cypress/integration/accessibility/tabbing-order.e2e-spec.ts

```diff
@@ -89,6 +90,18 @@ context('Tabbing order - tests do require user to be logged in', () => {
     cy.saveLocalStorage();
   });
 
+  describe('Order History', () => {
+    it('should allow to navigate with tab key (no orders)', () => {
+      orderHistoryNoOrdersTabbingOrder(config.orderHistoryNoOrders);
+    });
+
+    it('should allow to navigate with tab key (with orders)', () => {
+      cy.window().then(win => win.sessionStorage.clear());
+      cy.requireLoggedIn();
+      orderHistoryWithOrdersTabbingOrder();
```

Comment:
you can use the second config here (mentioned above)

Reply:
Can't for the reason mentioned above